docs/configuration.rst

```diff
@@ -143,7 +152,16 @@ manager, declare the extensions you want to enable:
                     <stof-doctrine-extensions:entity-manager
                         id="default"
                         tree="true"
-                        timestampable="false"
+                        timestampable="false" <!-- not needed: listeners are not enabled by default -->
+                        translatable="false"
+                        blameable="false"
+                        sluggable="false"
+                        loggable="false"
+                        ip_traceable="false"
+                        sortable="false"
+                        softdeleteable="false"
+                        uploadable="false"
+                        reference_integrity="false"
```

Comment:
XML attributes should use a dash instead of an underscore, to follow the usual XML convention (symfony/config is handling that normalization automatically).

As we don't have an XSD in this bundle, both `_` and `-` will work, but the expected one (the one we would define in XSD if we were adding it) is the `-`.